static/css/spaceUtilisation/grid.css

```diff
@@ -0,0 +1,102 @@
+* {
```

Comment:
It would be good keeping the units consistent. There's a fair mix of viewport, percentage, ems, pixels... in there.